build.gradle.kts

```diff
@@ -267,7 +267,7 @@ tasks.register<Sync>("replaceTokens") {
     val git = org.ajoberstar.grgit.Grgit.open(mapOf("currentDir" to project.rootDir))
     val revision = git.head().id
 
-    val versionFull = "$versionMajor.$versionMinor.$versionSubminor-SNAPSHOT"
+    val versionFull = "$versionMajor.$versionMinor.$versionSubminor"
```

Comment:
Should we use variable "version" from line 38?

Reply:
seems like that would be a better option.